### DISCUSSION TITLE
Jump Link Consideration on Documentation Site

### DISCUSSION THREAD

**Discussion by bennettcc** [NCIOCPL/ncids#1373] [NCIDS Launch]
Submitted by Alex during Doc site review: 

The anchor-jump link functionality is inconsistent in this page https://designsystem-dev.cancer.gov/develop/components/hero#package (#package is the only heading/section that has the functionality). Unsure if this is intentional, in some pages we anchor every heading (regardless of H Level). 

Same here: 
- https://designsystem-dev.cancer.gov/develop/components/list  
- https://designsystem-dev.cancer.gov/develop/components/skipnav  

What consistency should these pages have as it relates to jumplinks? What links/pages should use them?

**Comment by blilianyu** [comment]
We've updated the anchor links #1309. This could be closed?

cc: @monika-jaeger

**Comment by laurelthrash** [comment]
Discussed during 04/23 UX meeting that this has been addressed for the time being and the discussion can be closed out.
